src/hott/06-contractible.rzk.md

```diff
@@ -391,48 +393,48 @@ A type is contractible if and only if it has singleton induction.
 
 ```rzk
 #def ev-pt
-  (A : U)
-  (a : A)
-  (B : A -> U)
-  : ((x : A) -> B x) -> B a
+  ( A : U)
+  ( a : A)
+  ( B : A -> U)
+  : ( (x : A) -> B x) -> B a
```

Comment:
Perhaps a bit counterintuitively, I don't think it is necessary to have a space after the opening parenthesis on this line, as it does not help display any structure of the code.

Reply:
Are you referring specifically to Line 399?

Reply:
Yes

Reply:
Done